client/src/components/Rider/SelectedRide.jsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 
 const SelectedRide = props => {
-    
     return (
         <div className={"RiderBtmPanel"} id="SelectedRide">
             <div id="confirmationPage">
                 <span id="confirmationPageHeader">{!props.rideConfirmed ? 'Here are the ride\'s details:' : 'Ride details:'}</span>
                 <ul id="fullRideDetails">
-                    <li><span className="rideDetail">Pick up time: </span>{props.ride.time ? props.ride.time: 'Tuesday, Nov 18th at 9am'}</li>
-                    <li><span className="rideDetail">Pick up location: </span>{props.ride.start}</li>
-                    <li><span className="rideDetail">Drop off location: </span>{props.ride.end}</li>
+                    <li><span className="rideDetail">Pick up time: </span>{props.ride.date ? props.ride.date: 'Tuesday, Nov 18th at 9am'}</li>
+                    <li><span className="rideDetail">Pick up location: </span>{props.ride.pickUp}</li>
+                    <li><span className="rideDetail">Drop off location: </span>{props.ride.dropOff}</li>
                     <li><span className="rideDetail">Driver's Name: </span>{props.ride.driver ? props.ride.driver: 'Nate'}</li>
                 </ul>
```

Comment:
looks really nice. We could deconstruct 'ride' and 'rideConformed' from props here.